internal/kubectl/root.go

```diff
@@ -54,7 +55,7 @@ type Client interface {
 }
 
 func init() {
-	api.AddToScheme(scheme.Scheme)
+	utilruntime.Must(api.AddToScheme(scheme.Scheme))
```

Comment:
Can you explain this one a bit more?

Reply:
Just to ensure panic if an error occurs here. The function returns an error, and that gives a warning in my IDE if not handled (or ignored).

Reply:
Hmm, I might have just handled the error rather than pull in another dependency, I feel like that would be more obvious. LMKWYT and then I'll approve either way.

Reply:
The dependency is not "new". It is part of api-machinery. :wink But I have no strong feeling about this. Can also just ignore the error unintentionally - as it used to be.

Reply:
I mean there are lots of api-machinery packages we don't import. It's new to this file and that's added complexity. But it's nbd so I'll just approve.